sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,81 @@
+(5061, 2, 0, 0, 46, 0, 100, 0, 5061, 0, 0, 0, 0, 45, 1, 1, 0, 0, 0, 0, 10, 114829, 28617, 0, 0, 0, 0, 0, "Areatrigger - On Trigger - Set Data 1 1"),
+(5079, 2, 0, 0, 46, 0, 100, 0, 5079, 0, 0, 0, 0, 45, 1, 1, 0, 0, 0, 0, 10, 114830, 28617, 0, 0, 0, 0, 0, "Areatrigger - On Trigger - Set Data 1 1"),
+(5080, 2, 0, 0, 46, 0, 100, 0, 5080, 0, 0, 0, 0, 45, 1, 1, 0, 0, 0, 0, 10, 114831, 28617, 0, 0, 0, 0, 0, "Areatrigger - On Trigger - Set Data 1 1");
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` IN (5051, 5061, 5079, 5080);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(22, 1, 5051, 2, 0, 1, 0, 51966, 0, 0, 0, 0, 0, "", "SAI areatrigger 5051 triggers only if player has aura Scourge Disguise"),
+(22, 1, 5061, 2, 0, 1, 0, 51966, 0, 0, 0, 0, 0, "", "SAI areatrigger 5061 triggers only if player has aura Scourge Disguise"),
+(22, 1, 5079, 2, 0, 1, 0, 51966, 0, 0, 0, 0, 0, "", "SAI areatrigger 5079 triggers only if player has aura Scourge Disguise"),
+(22, 1, 5079, 2, 0, 2, 0, 39319, 1, 0, 0, 0, 0, "", "SAI areatrigger 5079 triggers only if player has item Scepter of Domination"),
+(22, 1, 5080, 2, 0, 1, 0, 51966, 0, 0, 0, 0, 0, "", "SAI areatrigger 5080 triggers only if player has aura Scourge Disguise");
+
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` IN (-114832, -114829, -114830, -114831);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-114832, 0, 0, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 11, 52089, 0, 0, 0, 0, 0, 21, 10, 0, 0, 0, 0, 0, 0, "Drakuramas Teleport Bunny 01 - On Data Set 1 1 - Cast 'Drakuramas Teleport Script 01'"),
+(-114829, 0, 0, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 11, 52239, 0, 0, 0, 0, 0, 21, 10, 0, 0, 0, 0, 0, 0, "Drakuramas Teleport Bunny 01 - On Data Set 1 1 - Cast 'Drakuramas Teleport Script 02'"),
+(-114830, 0, 0, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 11, 52676, 0, 0, 0, 0, 0, 21, 10, 0, 0, 0, 0, 0, 0, "Drakuramas Teleport Bunny 01 - On Data Set 1 1 - Cast 'Drakuramas Teleport Script 03'"),
+(-114831, 0, 0, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 11, 52089, 0, 0, 0, 0, 0, 21, 10, 0, 0, 0, 0, 0, 0, "Drakuramas Teleport Bunny 01 - On Data Set 1 1 - Cast 'Drakuramas Teleport Script 01'");
+
```

Comment:
The teleport effect spell is linked with the teleport spell for 52089, 52676
``` sql
SELECT * FROM spell_linked_spell WHERE spell_trigger IN (52089, 52091, 52239, 52676, 52240,52677);

Reply:
'Spell only hits player' as compared to hitting an npc?

Reply:
Maybe the condition is wrong.

Reply:
Yes it must hit the npc:
```
ServerToClient: SMSG_SPELL_START (0xDB2F) Length: 31 ConnIdx: 0 Time: 06/10/2011 16:57:29.000 Number: 110863
Caster GUID: Full: 0xF1306FC900000671 Type: Creature Entry: 28617 Low: 1649
Caster Unit GUID: Full: 0xF1306FC900000671 Type: Creature Entry: 28617 Low: 1649
Cast Count: 0
Spell ID: 52089 (52089)
Cast Flags: 10 (HasTrajectory, Unknown3)
Time: 0
Target Flags: 0 (Self)

Reply:
That's a sniff, right?
What does it say about 52091?

Reply:
Casted by the player on him self:
``` sql
ServerToClient: SMSG_SPELL_START (0xDB2F) Length: 29 ConnIdx: 0 Time: 06/10/2011 16:38:02.000 Number: 80387
Caster GUID: Full: xxxxxxxx Type: Player Low: xxxxxxx
Caster Unit GUID: Full: xxxxxxxxxxxx Type: Player Low: xxxxxx
Cast Count: 0
Spell ID: 52091 (52091)
Cast Flags: 2 (HasTrajectory)
Time: 0
Target Flags: 0 (Self)